docs/src/2q_and_edb_repositories.md

```diff
@@ -109,16 +109,15 @@ cluster_vars:
     - products/bdr3/release
 ```
 
-This example will install the pglogical3 and bdr3 release repositories.
-On Debian and Ubuntu systems, it will use the APT repository, and on
-RedHat systems, it will use the YUM repository.
+This example installs the pglogical3 and bdr3 release repositories.
+On Debian and Ubuntu systems, it uses the APT repository, and on
+RedHat systems, it uses the YUM repository.
```

Comment:
Should YUM be capitalized in this case? It isn't in the earlier cases but I wasn't sure if the repository name should be capped.

Reply:
both yum and apt should be treated the same I don't think it needs to be capped, yum repository might even be replaced by rpm repository

Reply:
```suggestion
RedHat systems, it uses the yum repository.
```